src/python/pants/pantsd/pailgun_server.py

```diff
@@ -270,13 +273,16 @@ def yield_and_release(time_waited):
         yield
     else:
       # We have to wait for another request to finish being handled.
-      NailgunProtocol.send_stderr(request, "Another pants invocation is running. Will wait {} for it to finish before giving up.\n".format(
+      self._send_stderr(request, "Another pants invocation is running. Will wait {} for it to finish before giving up.\n".format(
         "forever" if self._should_poll_forever(timeout) else "up to {} seconds".format(timeout)
       ))
+      self._send_stderr(request, "If you want to have multiple pants invocations at once, please "
+                                 "press Ctrl-C and run with PANTS_ENABLE_PANTSD=False in the "
+                                 "environment.")
       while not self.free_to_handle_request_lock.acquire(timeout=user_notification_interval):
         time_polled += user_notification_interval
```

Comment:
Would you mind increasing the user notification interval to 5 seconds here as well?